### PR TITLE
0.25.1

### PR DESCRIPTION
**Pull request by dhatch**

### CODE REVIEW

File: docs/content/any/project/changelogs/2022-01-06.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+title: Release 2022-01-06
+menuTitle:2022-01-06
```
Comment (patrickod) [inline docs/content/any/project/changelogs/2022-01-06.md:3]:
```suggestion
menuTitle: 2022-01-06 
```
should appease the YAML parsing exceptions in the docs checks GH actions.

**Review by patrickod — COMMENTED** [review]
<accidentally commented instead of reviewed - repeated below>

**Review by patrickod — APPROVED** [review]
🚢 - suggested a YAML fix to unblock the docs GH actions.

File: languages/python/sqlalchemy-oso/tests/test_post_relationship.py [hunk 2]
```diff
@@ -802,6 +802,35 @@ def test_partial_isa_with_path(session, oso, tag_nested_many_many_test_fixture):
...
+        """
+    )
+
+    user = tag_nested_many_many_test_fixture["user"]
+    posts = session.query(Post).filter(
+        authorize_model(oso, user, "read", session, Post)
+    )
+    print_query(posts)
+    # Should only get posts created by user.
+    # TODO TEST HANGS
```
Comment (gkaemmer) [inline languages/python/sqlalchemy-oso/tests/test_post_relationship.py:826]: Does this mean the test hangs?

File: languages/python/sqlalchemy-oso/tests/test_post_relationship.py [hunk 3]
```diff
@@ -802,6 +802,35 @@ def test_partial_isa_with_path(session, oso, tag_nested_many_many_test_fixture):
...
+        """
+            allow(user, _, post: Post) if
+                check(user, post) and user.name == "user";
+
+
+            check(user: User, post: Post) if
+                post.created_by = u and
+                check(user, u.tag);
+            check(_: User, tag: Tag)      if tag.is_public;
+            #check(_: User, nottag: Post)      if nottag.bad;
```
Comment (gkaemmer) [inline languages/python/sqlalchemy-oso/tests/test_post_relationship.py:816]: Delete?

**Review by gkaemmer — APPROVED** [review]
I think this looks good, the new test seems a bit hacky but that's probably okay.

**Comment by dhatch** [comment]
I removed these tests. It's hanging. Seems like it's still a bug, I added it by accident.

**Pull request merged**